src/views/Community/components/MemberCard/index.tsx

```diff
@@ -2,39 +2,39 @@ import Gravatar from '@ferlab/ui/core/components/Gravatar';
 import { Card, Space, Typography } from 'antd';
 import { DEFAULT_GRAVATAR_PLACEHOLDER } from 'common/constants';
 import Highlighter from 'react-highlight-words';
-import { Link } from 'react-router-dom';
-import { TUser } from 'services/api/user/models';
+// import { Link } from 'react-router-dom';
```

Comment:
remove uneeded comments pls

Reply:
I will keep that one, it will be re-enable soon